attestation/policyverify/policyverify.go

```diff
@@ -0,0 +1,223 @@
+// Copyright 2023 The Witness Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package policyverify
+
+import (
+	"crypto"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/testifysec/go-witness/attestation"
+	"github.com/testifysec/go-witness/cryptoutil"
+	"github.com/testifysec/go-witness/dsse"
+	"github.com/testifysec/go-witness/log"
+	"github.com/testifysec/go-witness/policy"
+	"github.com/testifysec/go-witness/slsa"
+	"github.com/testifysec/go-witness/source"
+	"github.com/testifysec/go-witness/timestamp"
+)
+
+const (
+	Name = "policyverify"
+	Type = slsa.VerificationSummaryPredicate
+)
+
+var (
+	_ attestation.Subjecter = &Attestor{}
+	_ attestation.Attestor  = &Attestor{}
+)
+
+type Attestor struct {
```

Comment:
Does this need to be registered in an init() like the others? I know the `RunType` `ExecuteRunType` already is registered, but they are registered to different Attestor Types. 

This would be missing from `FactoryByType` for example if I'm not mistaken... is it expected to be accessed in this way?

Reply:
I am right in saying that https://github.com/in-toto/go-witness/pull/55/files#diff-f12a0e5e9e75cb021efdaf70898369096fe67b0aa85fc890b709c54e9183e9d3R48 satisfies this @mikhailswift ? If so we can go ahead and resolve this comment. Thanks @BHunter2889 😄